src/qq.jl

```diff
@@ -11,17 +11,34 @@ function qqbuild(x::AbstractVector, y::AbstractVector)
     return QQPair(qx, qy)
 end
 
+
+"""
+Generates a sequence of probability points of length `n`:
+
+``
+(k − a)/(n + 1 − 2a), k ∈ 1, ..., n
+``
+
+`a` should be ∈ [0,1]. See the references listed here:
+https://en.wikipedia.org/wiki/Q%E2%80%93Q_plot#Heuristics
```

Comment:
Some style changes to make it consistent:
````suggestion
    ppoints(n::Int, a::Real=0.5)

Generate a sequence of probability points of length `n`:

```math
(k − a)/(n + 1 − 2a), k ∈ 1, ..., n
```

`a` should be a number in ``[0, 1]``.

## References

https://en.wikipedia.org/wiki/Q%E2%80%93Q_plot#Heuristics
````